### DISCUSSION TITLE
Issues with installation of pycaret. Please Help.

### DISCUSSION THREAD

**Discussion by fookaixiong** [pycaret/pycaret#2785] [Q&A]
Working on Kaggle notebook.
Previously used pycaret without issues, but have some issues at the moment.
Tried uninstalling and re-installing pycaret, but no success.

Tried: 
!pip install pycaret
Error: ERROR: Cannot uninstall 'llvmlite'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.

After which i uninstalled and tried: 
!pip install pycaret--ignore-installed llvmlite
Error: Refer to attached image
![pycaret error](https://user-images.githubusercontent.com/108969905/180014013-19579e3a-d734-4407-854f-e80db6cdc4d1.PNG)

Not sure what to do, please advise.

Thank you

**Comment by ngupta23** [comment]
Please see if this helps: https://github.com/pycaret/pycaret/issues/1710#issuecomment-1087909205

**Reply by fookaixiong** [reply]
Hi, thanks for the reply.

I have tried both, however both does not seem to solve the issue. I have attached image below for reference.

!pip install --ignore-installed pycaret-ts-alpha
![pycaret error 3](https://user-images.githubusercontent.com/108969905/180021351-6cb478c6-5c48-44a6-9c3d-00d2f7810920.PNG)


!pip install --ignore-installed pycare
![pycaret error 2](https://user-images.githubusercontent.com/108969905/180021380-e98e8322-221a-4b86-a4fd-44c362d7f7f2.PNG)

Thank you

**Reply by ngupta23** [reply]
One more try - try the latest prerelease version of 3.0.0rc

`pip install -U --ignore-installed --pre pycaret`

**Reply by fookaixiong** [reply]
Awesome it works Thank you!
